src/main/scala/firrtl/passes/AnnotateValidMemConfigs.scala

```diff
@@ -58,7 +57,7 @@ case class SRAMConfig(
     val fieldMap = getClass.getDeclaredFields.map { f => 
```

Comment:
I am more familiar and more comfortable with braces being preceded by a space, but not parens.  Mostly I'd here I am trying to get the source to pass the standard style guide without warning.  I don't have strong feelings about this but I think it makes code blocks stand out, and has a symmetry with match {